packages/extension/src/pages/chat/index.tsx

```diff
@@ -61,6 +61,11 @@ const ChatView = () => {
 
   const [userChats, setUserChats] = useState<MessageMap>({});
   const [inputVal, setInputVal] = useState("");
+  const [open, setIsOpen] = useState(true);
+  const [messagingPublicKey, setMessagingPublicKey] = useState("");
+  const [openDialog, setIsOpendialog] = useState(false);
```

Comment:
openDialog && open why do we need two diffrent states, cant we just achieve it with openDialog

Reply:
I am fixing this.